example_analysis/run_entire_analysis_slurm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#SBATCH --job-name=all   # Job name
+#SBATCH --partition=20                   # Partition name
+#SBATCH --ntasks=1                       # Run a single task
+#SBATCH --cpus-per-task=1               # Single CPU for the controller job
+#SBATCH --mem=4G                        # Memory for the controller job
+#SBATCH --time=72:00:00                 # Time limit (hrs:min:sec)
+#SBATCH --output=slurm_output/main/all-%j.out  # Standard output log
+
+# Set the path to the main Snakefile and the config file
+BRIEFLOW_PATH="../"
+CONFIG_FILE_PATH="config/config.yml"
+PROFILE_FILE_PATH="slurm/"
+RULEGRAPH_FILE="../images/brieflow_rulegraph.png"
+
+# Create slurm output directories
+mkdir -p slurm_output/rule
+
+# Activate conda environment (adjust path as needed)
+source ~/.bashrc
+conda activate brieflow_workflows
+
+# Generate a rulegraph of the Snakefile
+# NOTE: Uncomment when needed, takes extra computation
+# snakemake --snakefile "$SNAKEFILE" --configfile "$CONFIG_FILE" --rulegraph | dot -Gdpi=100 -Tpng -o "$RULEGRAPH_FILE"
+
+# Run Snakemake with the specified Snakefile and config file
+snakemake --use-conda --executor slurm \
+    --workflow-profile "$PROFILE_FILE_PATH" \
+    --snakefile "${BRIEFLOW_PATH}workflow/Snakefile" \
+    --configfile "$CONFIG_FILE_PATH"
```

Comment:
```suggestion
    --configfile "$CONFIG_FILE_PATH"
    
```

Reply:
Changed in latest [commit](https://github.com/cheeseman-lab/brieflow/pull/18/commits/485b5e89ea72815b1fdb0a3c596c547425a9c232)